pkg/capi/infrastructure/aws.go

```diff
@@ -83,7 +83,7 @@ func NewAWSDeployOptions() *AWSDeployOptions {
 		NodeMachineType:         "t3.large",
 		NodeIAMProfile:          "CAPI_AWS_Worker",
 		CloudProviderVersion:    "v1.20.0-alpha.0",
```

Comment:
this is not needed anymore

Reply:
remove it completely?

Reply:
it also depends I guess on the template used... different projects use different templates, so I will probably leave it for now

Reply:
yes, not used in the new templates, since the aws ccm is now generated off helm charts

Reply:
yep, but this project is used in multiple projects, including those using older templates... so if I drop I might inadvertently break them